spec.md

```diff
@@ -398,6 +398,34 @@ Example error response (invalid `id` provided):
 
 ### Resource Data Model
 
+A resource is a named (addressable) blob stored in a Space, with metadata.
+The data model is derived from [W3C FileAPI: File 
+Interface](https://w3c.github.io/FileAPI/#file-section), but with the addition
+of a few crucial properties.
+
+In similar storage systems, a resource is called "File", "Object", "Document",
+"Row", "Graph", and so on.
+
+Resource properties:
+
+* `id`
+* `contents` (Blob)
```

Comment:
If we use `contents`, I want it to be conformant with as2:contents. https://www.w3.org/TR/activitystreams-vocabulary/#dfn-content

Because a resource may have blobs of many media types, I would prefer not to list it here.

consider that the blob could be 100Mb and it would make a JSON object really big.

For naming the relationship between a resource and its many representations, I think maybe like `representations` might be a better name to use.

(also do we really even need to define this here? there may be different properties in different contexts, which we can illustrate in HTTP API Docs)

Reply:
Also good point, we don't need it here.